### PR TITLE
Refactor: moesif-laravel dependency to 1.4.7

### PR DESCRIPTION
**Pull request by keyur9**
Refactor: moesif-laravel dependency to 1.4.7

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -1583,7 +1583,7 @@
                 "logging",
                 "psr-3"
             ],
-            "time": "2019-11-13T10:00:05+00:00"
+            "time": "2019-12-20T14:15:16+00:00"
```
Comment (dkm199) [inline composer.lock]: is time really needed here?

Comment (dkm199) [reply composer.lock]: doesn't make sense to update every time u update release if not needed?

Comment (keyur9) [reply composer.lock]: `composer.lock` is an autogenerated file when we install/update dependencies. We are not manually updating it.

Comment (dkm199) [reply composer.lock]: ok.

**Pull request merged**